app_dart/lib/src/service/cache_service.dart

```diff
@@ -55,6 +55,7 @@ class CacheService {
     int attempt = 1,
     Future<Uint8List> Function() createFn,
     Duration ttl = const Duration(minutes: 1),
+    bool flushCache = false,
   }) async {
```

Comment:
This function becomes too smart that it has three concerns now, indicators are the lengthy testing setups. 

It's better to separate the concerns into `get()`, `create()` and `flush()`, and when a smart business logic function is needed, compose that with these dump functions.

Reply:
Good point. I moved this extra flush logic to a separate function. I made available the `purge` function with flushes a value from the cache, and had `CacheRequestHandler` use that instead.

Reply:
Nit: I would split get() and create() as well. Doesn't need to address in this PR though.